grype/presenter/models/models_helpers.go

```diff
@@ -29,6 +29,17 @@ func GenerateAnalysis(t *testing.T, scheme syftSource.Scheme) (match.Matches, []
 	return matches, packages, context, NewMetadataMock(), nil, nil
 }
 
+func GenerateAnalysisWithIgnoredMatches(t *testing.T, scheme syftSource.Scheme) (match.Matches, []match.IgnoredMatch, []pkg.Package, pkg.Context, vulnerability.MetadataProvider, interface{}, interface{}) {
+	t.Helper()
+
+	packages := generatePackages(t)
+	matches := generateMatches(t, packages[0], packages[0])
+	ignoredMatches := generateIgnoredMatches(t, packages[1])
```

Comment:
`generatePackages()` returns 2 packages. I wanted to use one package for generating ignored matches, so I passed the first package twice to `generateMatches()`. This feels a bit strange, but I'm not sure how to change it.